docs/projects/templates/project-proposal-pr-template.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Project Proposal
+about: A suggestion for a new project to work on
+labels: "💬 talk: discussion"
+title: "<Replace this with actual title>"
+---
+
+<!--
```

Comment:
Can these comment blocks be moved inside of the sections they're relevant to? They feel displaced at the moment :sweat_smile:

Reply:
Ah yep, they do look quite odd. I think before our auto formatting they were right up against the section titles without any newline 😅